src/actions/index.cy-spec.js

```diff
@@ -0,0 +1,45 @@
+import * as types from '../constants/ActionTypes'
+import * as actions from './index'
+
```

Comment:
compare to https://github.com/cypress-io/cypress-example-todomvc-redux/blob/master/src/actions/index.spec.js